files/ja/web/api/audiobuffersourcenode/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: AudioBufferSourceNode
 slug: Web/API/AudioBufferSourceNode
+browser-compat: api.AudioBufferSourceNode
```

Comment:
According to #7412, the `browser-compat` metadata should be deleted from translated contents, but why is this line added?

Reply:
Ah, I might have seen this in the English content, not sure to be honest :)